src/test/java/hudson/plugins/cobertura/RatioTest.java

```diff
@@ -21,4 +21,8 @@ final void assertRatio(Ratio r, float numerator, float denominator) {
     public void testParseValue() throws Exception {
         assertRatio(Ratio.create(1,2), 1.0f, 2.0f);
     }
+
+    public void testGetPercentage() {
+        assertEquals(99,Ratio.create(249,250).getPercentage());
+    }
```

Comment:
I'd like to see a couple more unit tests here - both 99 and 100 would be could. Also I think that we need to verify the printed string

Reply:
Ok, I'll add more test code in there. Thanks for your comment.